astropy/utils/console.py

```diff
@@ -359,13 +328,6 @@ def color_print(*args, end="\n", **kwargs):
         write(end)
 
 
-def strip_ansi_codes(s):
```

Comment:
Hmm I wonder what this was used for originally but looks okay to remove.

Reply:
This function was introduced in cb9e50d027066238aa3a98c03dddb05ac1e5a65b (AuthorDate: Mon Dec 5 21:03:47 2011 -0500) and used in a test:
```
$  git grep --files-with-matches strip_ansi_codes cb9e50d027066238aa3a98c03dddb05ac1e5a65b
cb9e50d027066238aa3a98c03dddb05ac1e5a65b:astropy/io/vo/tests/vo_test.py
cb9e50d027066238aa3a98c03dddb05ac1e5a65b:astropy/utils/console.py
```
All uses of it were removed in 46576785b98a43945c753f336b4eb6ecb844b510 (AuthorDate: Tue Dec 6 11:12:41 2011 -0500):
```
$ git grep --count strip_ansi_codes 46576785b98a43945c753f336b4eb6ecb844b510
46576785b98a43945c753f336b4eb6ecb844b510:astropy/utils/console.py:1
```

Reply:
Wow, back in #101 . Thanks for checking!